lib/vdsm/virt/migration.py

```diff
@@ -151,6 +152,9 @@ def __init__(self, vm, dst='', dstparams='',
         self._consoleAddress = consoleAddress
         self._dstqemu = dstqemu
         self._encrypted = encrypted
+        if parallel == self._PARALLEL_CONNECTIONS_DISABLED_VALUE:
+            parallel = None
```

Comment:
reading the [wiki page](https://www.ovirt.org/develop/release-management/features/virt/parallel-migration-connections.html) again, you wrote that if the number of connections is set to 1 we should rather not use parallel connections but here it's disabled only in case of null or 0 - do we expect the engine not to specify parallel=1 ?

Reply:
Yes, it's up to Engine and not Vdsm to decide.  If we ever want to permit parallel migrations with a single connection, it's better when the support in Vdsm is already present.

Reply:
ack